other-docs/guides/upgrading/v7.md

```diff
@@ -0,0 +1,141 @@
+
+The latest version of S3 Uploads has been refactored to use namespaces instead of prefixed class names and functions.
+
+If you have any code referencing the class `S3_Uploads` this will need to be updated to `S3_Uploads\Plugin`. Additionally any other references to classes starting with `S3_Uploads` should be updated to use `S3_Uploads` as the namespace, for example `S3_Uploads_Local_Stream_Wrapper` becomes `S3_Uploads\Local_Stream_Wrapper`. Any functions prefixed with `s3_uploads_*` have also been moved into the `S3_Uploads` namespace, for example `s3_uploads_enabled()` is now `S3_Uploads\enabled()`.
+
+All constants remain unchanged.
+
+### Altis Experiments
+
+The experiments feature of native analytics has been merged into the main native analytics codebase. This means any references in your code to the namespace `Altis\Experiments` should be updated to `Altis\Analytics\Experiments`.
+
+Additionally the `Altis\Analytics\Experiments\register_goal()` function accepts a different set of arguments than it did in Altis v6.
+
+
+## Headline Features
+
+### WordPress 5.7 "Esperanza"
+
+The latest CMS module includes WordPress 5.7. Some of the highlights of this release are:
+
+- Improved Reusable Block editing in the block editor. Reusable Blocks can now be fully edited inline in a post and saved separately to the post they are embedded in.
+- A [new Robots API](https://make.wordpress.org/core/2021/02/19/robots-api-and-max-image-preview-directive-in-wordpress-5-7/) provides control over the robots meta tag and search engine previews
+- Lazy loading for iframes
+- Ability to send password reset links to users
+
+Be sure to read the [release post](https://wordpress.org/news/2021/03/esperanza/) and in addition the full [WordPress 5.7 Field Guide can be found here](https://make.wordpress.org/core/2021/02/19/robots-api-and-max-image-preview-directive-in-wordpress-5-7/).
+
+### Experience Block Insights
+
+Continuing from previous improvements to tracking the performance of Experience Blocks in v6, Altis v7 adds a dedicated dashboard for showing all Experience Blocks across the site with options for sorting by views and conversion rate.
+
+When editing blocks warnings are shown to authors if their selected conversion goals can not be tracked, and if they are missing fallback content.
+
+In addition there is a new, more detailed overview of Experience Block data that more clearly shows audience coverage, views and conversion rate changes over time and also detailed breakdowns of how each variant is performing compared to the fallback.
+
+On the software level several new REST API endpoints have been added for fetching blocks and block anlaytics data, along with a background post type that can be used for storing persistent data.
+
+### Authorship
+
+Altis now supports adding multiple authors to posts as well as managing guest authors, either as full or partial users.
+
+This feature was funded by and built in collaboration with Siemens.
```

Comment:
Do we have a docs page for this?

Reply:
yeah good shout, I'll link to that above but thinking about it now I might leave this sentence at the bottom out, there's still a few things being agreed there

Reply:
```suggestion
[Full documentation for the Authorship feature including how to enable it can be found here](docs://cms/authorship/).
```